src/domain.c

```diff
@@ -481,6 +576,14 @@ static LDAPMod ** fill_attributes(LDAPAttribute_t **entry_attrs, TALLOC_CTX *tal
     attrs[attr_count] = NULL;
 
     return attrs;
+
+    error_exit:
+        if (attrs)
+        {
+            ld_talloc_free(attrs, error_exit);
```

Comment:
Do not use error_exit mark in error_exit statement. For explanation Look for description of first similar issue at the beginning of the file.